### PR TITLE
chore: allow specifying rosetta location in pack

### PR DESCRIPTION
**Pull request by MrArnoldPalmer**
Allow specifying custom location for jsii-rosetta in pack.sh.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: pack.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ export NODE_OPTIONS="--max-old-space-size=4096 ${NODE_OPTIONS:-}"
 root=$PWD
 
 PACMAK=${PACMAK:-jsii-pacmak}
+ROSETTA=${ROSETTA-jsii-rosetta}
```
Comment (skinny85) [inline pack.sh]: Shouldn't there be some default here?

Comment (MrArnoldPalmer) [reply pack.sh]: Yes was missing, fixed. 👍

**Comment by MrArnoldPalmer** [comment]
Holding merge on this, pkglint error is some environment problem. Will likely remove that and just leave the pack.sh addition.

File: pack.sh [hunk 2]
```diff
@@ -8,6 +8,7 @@ export NODE_OPTIONS="--max-old-space-size=4096 ${NODE_OPTIONS:-}"
 root=$PWD
 
 PACMAK=${PACMAK:-jsii-pacmak}
+ROSETTA=${ROSETTA:-jsii-rosetta}
```
Comment (RomainMuller) [inline pack.sh]: Kinda sorta thinking I'd rather those be prefixed with `JSII_`, but not a deal breaker.

**Pull request merged**